packages/@aws-cdk/aws-backup/rosetta/default.ts-fixture

```diff
@@ -0,0 +1,26 @@
+// Fixture with packages imported, but nothing else
+import { Duration, RemovalPolicy, Stack } from '@aws-cdk/core';
+import { Construct } from 'constructs';
+import * as backup from '@aws-cdk/aws-backup';
+import * as dynamodb from '@aws-cdk/aws-dynamodb';
+import * as events from '@aws-cdk/aws-events';
+import * as kms from '@aws-cdk/aws-kms';
+import * as lambda from '@aws-cdk/aws-lambda';
+import * as sns from '@aws-cdk/aws-sns';
+
+declare const fn: lambda.Function;
+declare const myKey: kms.Key;
+declare const myCoolConstruct: Construct;
+declare const myTable: dynamodb.Table;
+declare const myTopic: sns.Topic;
+declare const myVault: backup.BackupVault;
+declare const otherVault: backup.BackupVault;
+declare const plan: backup.BackupPlan;
```

Comment:
create a separate fixture that provides a plan so that the default is not polluted